lib/mapper.js

```diff
@@ -164,14 +168,16 @@ function createBaseTrackProperties(track, settings) {
   if (settings.prefixProperties) {
     properties = prefix(track.event(), properties);
   }
-
-  return extend(
-    properties,
-    {
+  // Only apply userAgent property on `visited site` event
+  if (track.event() === 'visited site') {
+    properties._userAgent = userAgent || track.proxy('context.userAgent');
+  }
```

Comment:
Percy is correct. Seems there was a misunderstanding; _userAgent was not implemented, only _ip.
If this is a requirement it's also a straightforward add.